core-bundle/src/Resources/contao/classes/Hybrid.php

```diff
@@ -296,6 +298,27 @@ protected function isHidden()
 		return true;
 	}
 
+	private function overwriteConfigFromParent(): void
+	{
+		if ($this->objParent->formSettings ?? false)
+		{
+			// Overwrite form configuration
+			foreach ($this->objParent->row() as $key => $value)
+			{
+				if (!str_starts_with($key, 'form_'))
```

Comment:
shouldn't this check the DCA config for overwrite-fields instead?

Reply:
Yes, you are right. Fixed in https://github.com/contao/contao/pull/3252/commits/38faeb9e25e2f2d2e90f5d74831384a68d36e8cb